### PR TITLE
doc: add minutes for meeting July 16 2024

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2024-07-16.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      * Richard
+        * only release container machines. Ubuntu 1804 - docker host
+        * does not need to be 1804
+        * Should be ansible playbook for it
+
+* macOS refuses to start `node` when downloaded from the tarball [#3538](https://github.com/nodejs/build/issues/3538)
+  * No update
+
+* Discuss state of r2 migration on Build WG meeting [#3508](https://github.com/nodejs/build/issues/3508)
+  * A change was merged, broke release workflow.
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2024-07-16.md:70]:
```suggestion
  * A change was merged, broke release workflow. It was partially backed out to allow the security release to happen. 
  * Subsequent fix has been applied but looks like there might be an issue promoting nightlies in R2.
```

File: doc/meetings/build-wg-meeting-2024-07-16.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* No announcements this week
+
+### nodejs/build
+
+* Planning/requirements for Node.js 23 [#3807](https://github.com/nodejs/build/issues/3807)
+  * Richard need to decide what infra changes we need/want. Good to have in place for Sep
+    to be ready for 23 in Oct.
+    * for example proposal is to move up to gcc 12
+    * only question would be for 32 bit arm binaries, did find that moving on this shifted glibc level
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2024-07-16.md:29]:
```suggestion
    * only question would be for 32 bit arm binaries, did find that moving on this shifted libstdc++ level
```

**Comment by mhdawson** [comment]
@nodejs/build any reviewers?

**Comment by mhdawson** [comment]
@nodejs/build

**Comment by mhdawson** [comment]
@nodejs/build would be great to get a review so that this can land.

**Comment by mhdawson** [comment]
@richardlau could you take another look and approve so we could get it landed?

**Review by richardlau — APPROVED** [review]

**Pull request merged**